Sources/Fuzzilli/Modules/Statistics.swift

```diff
@@ -75,9 +78,23 @@ public class Statistics: Module {
             self.ownData.totalExecs += 1
             self.currentExecs += 1
         }
-        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { _ in
+        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { ev in
             self.ownData.interestingSamples += 1
             self.ownData.coverage = fuzzer.evaluator.currentScore
+
+            if !ev.program.runtimeTypes.isEmpty {
+                self.ownData.interestingSamplesWithTypes += 1
+            }
+
+            if ev.typeCollectionOutcome != nil {
+                self.ownData.typeCollectionAttempts += 1
+            }
+
+            if ev.typeCollectionOutcome == .timedOut {
```

Comment:
Maybe also use the `case .timedOut` syntax here for consistency? But if you use a new `TypeCollectionResult` enum, this code will become simpler anyway